include/MidiWinMM.h

```diff
@@ -28,8 +28,11 @@
 #include "lmmsconfig.h"
 
 #ifdef LMMS_BUILD_WIN32
-#include <windows.h>
+
+// clang-format off
```

Comment:
I wonder if we should tell clang-format here: If you have windows.h, put it into a separate category above the other headers. It would work, but I wonder if it looks strange to have windows.h in a separate header group...